### PR TITLE
Update clue

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

**Comment by rpiaggio** [comment]
Tests are failing for the moment because of dependency in `lucuma-graphql-routes`, which I'll be updating shortly.

File: modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala [hunk 1]
```diff
@@ -55,7 +55,7 @@ object ItcClient {
 
         val callOut: F[SpectroscopyResult] =
           for {
-            r  <- http.request(SpectroscopyQuery)(input)
+            r  <- http.request(SpectroscopyQuery).withInput(input)
```
Comment (cquiroz) [inline modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala:58]: I tried to port this but I totally missed how to fix this line

**Review by cquiroz — APPROVED** [review]
LGTM

File: build.sbt [hunk 2]
```diff
@@ -18,7 +18,7 @@ val slf4jVersion                = "2.0.7"
 val log4catsVersion             = "2.5.0"
 val monocleVersion              = "3.2.0"
 val munitCatsEffectVersion      = "1.0.7"
-val graphQLRoutesVersion        = "0.5.11"
+val graphQLRoutesVersion        = "0.6.0"
```
[the remark this answers is not in the document]
Comment (rpiaggio) [reply build.sbt:21]: No, it was still release with `0.10.3`

File: modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/client/shared/src/main/scala/lucuma/itc/client/ItcClient.scala:58]: Actually, I just added this in clue so that this is easier now.

**Pull request merged**